Dockerfile.openhabian

```diff
@@ -0,0 +1,56 @@
+FROM minimum2scp/systemd-buster
+#FROM balenalib/intel-nuc-debian-node:latest-buster
+#FROM balenalib/raspberry-pi2-debian-node:latest-buster
+
+
+ENV container docker
+ENV INITSYSTEM on
+ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE=DontWarn
+
+RUN apt-get update -qq \
+    && apt-get install -y -qq --no-install-recommends systemd systemd-sysv git wget python python-pip python3 python3-pip apt-utils jq virt-what \
+    && rm -rf /var/lib/apt/lists/*
+RUN git clone https://github.com/bats-core/bats-core.git && \
+    cd bats-core && \
+    ./install.sh /usr/local
+
+# We never want these to run in a container
+# Feel free to edit the list but this is the one we used
+RUN systemctl mask \
+    dev-hugepages.mount \
+    sys-fs-fuse-connections.mount \
+    sys-kernel-config.mount \
+    display-manager.service \
+    getty@.service \
+    systemd-logind.service \
+    systemd-remount-fs.service \
+    getty.target \
+    graphical.target
+
+WORKDIR /
+COPY entry.sh /usr/bin/entry.sh
+COPY resin.service /etc/systemd/system/resin.service
+
+RUN systemctl enable /etc/systemd/system/resin.service
+
+RUN git clone https://github.com/gdraheim/docker-systemctl-replacement
+RUN cp -p /bin/systemctl /bin/systemctl.ORIG
+RUN cp docker-systemctl-replacement/files/docker/systemctl.py /bin/systemctl
+
+ENTRYPOINT ["/usr/bin/entry.sh"]
+
+RUN adduser openhabian --gecos "Openhabian,,," --disabled-password
+RUN echo "openhabian:openhabian" | chpasswd   
+RUN /bin/echo -n "Running on " && /usr/bin/arch
+
+COPY . /opt/openhabian/
+COPY openhabian.conf.dist /etc/openhabian.conf
```

Comment:
hm, we have a build-image/openhabian.conf, which defines usernames, java_arch, etc which might be also relevant for the test.... 
though it might be fine to use this settings from the .dist file.
btw: if openhabian.conf.dist is only for testing-docker, move it to the dedicated directory

Reply:
it wasn't me to introduce openhabian.conf.dist so I presume it has another use and I did not want to break that. Don't fully know though what it has been there for.
On build/*conf, I'm still struggling with "wrong" java in #684 but have not seen any other impact  to date.

Reply:
now sourced/applied java_arch, too